app/api/routes/user.py

```diff
@@ -1,8 +1,41 @@
-from fastapi import APIRouter
+from fastapi import APIRouter, HTTPException, status
+from models import UserCreate, User  
+import re
 
 router = APIRouter()
 
-# make sure to remove this function
-@router.get("/example")
-def example():
-    return {"message": "this is an example"}
+
+users_db = {}
+
+#Password regex
+password_regex = (
+    r'^(?=.*[A-Z])(?=.*[a-z])(?=.*[0-9])(?=.*[@#$%^&+=]).{8,}$'
+)
+
+@router.post("/users/", response_model=User, status_code=status.HTTP_201_CREATED)
+async def create_user(user: UserCreate):
+    #Validate password
+    if not re.match(password_regex, user.password):
```

Comment:
any input validation should be in the validation model, in UserCreateRequestModel

Reply:
I replace it by validator in UserCreateRequestModel, done.